### DISCUSSION TITLE
Is Eartrumpet an active project?

### DISCUSSION THREAD

**Discussion by tomber42** [File-New-Project/EarTrumpet#730] [Q&A] [answered]
Hi

Is Eartrumpet an active project where we can expect new releases and new features in the near future?

Greetings 
tomber

**Comment by riverar** [comment, marked as answer]
Yes! EarTrumpet has certainly matured over the past year or so, but we have a bunch of add-on work happening as you read this. You can see a sneak peek of that activity in https://github.com/File-New-Project/EarTrumpet/pull/624#issuecomment-770461880. A lot of the feature requests have been implemented via our EarTrumpet Actions add-on, stay tuned!
